### PR TITLE
BookStore: Styling Branch

### PR DESCRIPTION
**Pull request by Donmark2k**
### Highlights 🥇

**BookStore**

**Learning objectives**

- Use React Router.
- Use React components.
- Understand the concept of state management.
- Use store, actions, and reducers in React
- Understand the concept of state management.
- Connect React and Redux.
- Connect an existing API via Redux and use the data to display something in a UI.
- Apply React best practices and language style guides in code.

**What I Have Done:**

- Added linters. The [react & redux linter](https://github.com/microverseinc/linters-config/tree/master/react-redux).
- I create a git flow.
- I deleted all the boilerplate from Create React App (text, images, styles).
-  I added a reducer
-  I created a store 
-  I connect to API

**Followed Rules:**

- Made sure [no linter errors](https://github.com/microverseinc/linters-config)
- Followed [Git Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)
- Documented [professionaly](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
- Followed the list of [best practices for react & redux](https://github.com/microverseinc/linters-config/tree/master/react-redux).

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,5 +1,216 @@
...
+
+.bookContainer {
+  background-color: white;
+  margin: 1.438rem 7.5rem 1.688rem 7.5rem;
+  border: 1px solid #888;
+  padding: 1.438rem 1.5rem;
+  border-radius: 10px;
+  display: flex;
+  justify-content: space-between;
+}
```
Comment (TadesseDev) [inline src/App.css]: - [ ] Your design is almost exceptional @Donmark2k, But there are some minor details to check. As you can see from the Zeplin design the border for the `bookContainer` is not visible black, instead, it has a very thin box shadow which sounds like a border. This is to make sure you have the exact design clone.
### Your design. 
image
### Zeplin design
image

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Donmark2k 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## Highlights
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitFlow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @Donmark2k  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights 
:+1: no linter errors 
:+1: good styling

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Donmark2k** [comment]
Thank you

**Pull request merged**